data/incipias/incipias first contact.txt

```diff
@@ -0,0 +1,405 @@
+			`	You quickly explain what happened. From what you can tell, the Quarg seems relieved that you were not able to talk to them.`
+			label explanation
+			`	"You ran into a most peculiar situation. Surpassing the borders of their system is not yet a capability possessed by the Incipias." It tilts its head for a moment, considering what you just said. "It is our desire to help them, but it is yet premature for us to reveal ourselves to their species, and certainly premature for them to be interacting with independent interlopers such as yourself. I fear that we are impotent in this unfortunate turn of events."`
+			choice
+				`	"Then let me help them."`
+					goto initiative
+				`	"I understand. I'll leave them alone."`
+					goto declined
+				`	"How can you make such a cold-hearted decision?"`
+			`	"It is perhaps a laborious task for an adolescent species such as yourself to comprehend our ways, but we must place the continuation of species above the existence of individuals. If we reveal ourselves to them at this inopportune moment, the shock of such an advanced species appearing in their space will fan the flames of chaos."`
+			choice
+				`	"Then let me help them."`
+					goto initiative
+				`	"You can deal with the consequences later. Help them now!"`
+			`	The Quarg takes on a sour look. "Insisting you are. Understand that our own intervention is out of the question." It sighs. "However, I can foresee the likelihood of you making a rash and impetuous decision to contact this ship."`
+			choice
+				`	"Sure, let me do the work."`
+				`	"If you are too weak..."`
+			`	The Quarg rapidly clicks its teeth, before continuing.`
+			label initiative
+			`	"You lack the tools for such a mission. To my knowledge, no vessels capable of landing on gas giants have ever been developed by your kind. And you are certainly not able to make your intentions known to them in any meaningful sense."`
+			`	"But for the sake of the Incipias, I will assist you on this last point. If you cannot be dissuaded from your course of action, then it is best you are supplied a means of communication, so that your interactions may be on amicable terms. This one can produce you with a single one of our translators, which will allow you to relate your intentions to the species in their home system. Request that they sell you one of their ships, so that you are made capable to land on their tumultuous planets."`
+			`	It pauses before taking on a more serious tone. "However, you are forbidden from granting the young species any advanced technology of yours. If you concur with us on that point, we shall furnish you with a translator."`
+			label choice
+			choice
+				`	"If that's how I can help them, sure."`
+					goto yes
+				`	"Why do you not want me to sell my tech?"`
```

Comment:
```suggestion
				`	"Why do you not want me to give them my tech?"`
```
You might not sell them, you might just give them a Gridfire turret for fun for all the Quarg know.